Add more frameworks

mantoni opened the pull request:
This runs tests with Mochify and Jest and adds a short description to the readme.

mantoni committed:
Add mochify

mantoni committed:
Add jest

mantoni committed:
Improve README

mroderick reviewed (approved):
I have verified this locally 👍

mroderick commented:
How about adding examples for:

* https://github.com/substack/tape
* https://github.com/jasmine/jasmine

I don't know what else people use

mantoni committed:
Add jasmine

mantoni commented:
I've added Jasmine and added more description and a link to the `referee.add` docs. Adding Tape is not so easy because it requires the tests to be written differently. Tap has a BDD style API like Mocha, but needs an initial function call to install it.

mantoni committed:
Add tap

mantoni commented:
Also managed to add Tap with a helper file.

mroderick reviewed (commented):
Tap is a dependency, the rest are dev dependencies. Is that intentional?

mantoni commented:
Good catch. Fixed.
